setup.py

```diff
@@ -32,13 +32,27 @@
         r".*__version__ = '(.*?)'", re.S).match(v_file.read()).group(1)
 
 
-def read(fname):
-    """
-    Read filename.
 
-    :param str fname: name of a file to read
-    """
-    return open(os.path.join(here, fname)).read()
+try:
+    from pypandoc import convert
```

Comment:
There's no need for that https://bitbucket.org/pypa/pypi/issues/148/support-markdown-for-readmes (or check ianitor)